base/broadcast.jl

```diff
@@ -127,6 +131,7 @@ dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
     Base.copy_to_bitarray_chunks!(Bc, ((bind - 1) << 6) + 1, C, 1, min(bitcache_size, (length(Bc)-bind+1) << 6))
 
 @inline _broadcast_getindex(A, I) = _broadcast_getindex(containertype(A), A, I)
+@inline _broadcast_getindex(A::RefArrOrVal, I) = A.x
```

Comment:
I was thinking in limiting all this to `RefValue`. If we allow `Ref` and leave this definition of `_broadcast_getindex` as `A[]`, then we will have

```julia
broadcast(+, [[0,2], [1,3]], Ref([1,-1])) == [[1,3], [2,4]]
```

where we want it to be `[[1,1], [2,2]]`, which `broadcast(+, [[0,2], [1,3]], RefValue([1,-1]))` would correctly yield.